.clusterfuzzlite/build.sh

```
@@ -0,0 +1,134 @@
+            MatchingUtilities.executeValuePath(DATA, valuePath, equals, contains);
+        } catch (Exception ignored) {
+            // Expected — fuzzer explores error paths
+        }
+    }
+}
+EOF
+
+# ── Compile fuzz targets against the utility classes ──
+javac -source 21 -target 21 \
+    -cp "$OUT/classes" \
+    -d $OUT \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)